### PR TITLE
fix ambiguous duplicate bug

### PR DESCRIPTION
**Pull request by hcyang**
remove ambiguous entries if they are in the ground-truth labels

### CODE REVIEW

**Review by tsuwandy — APPROVED** [review]
Could you add a test?

**Review by daveta — APPROVED** [review]

File: packages/orchestratorlib/src/utility.ts [hunk 1]
```diff
@@ -4383,11 +4383,14 @@ export class Utility {
...
         const predictedScore: number =
           predictionStructureWithScore.predictionScoreStructureFoundation.labelsPredictedScore;
+        const labelsIndexes: number[] =
+          predictionStructureWithScore.labelsIndexes;
         const scoreArray: number[] =
           predictionStructureWithScore.predictionScoreStructureFoundation.scoreArray;
         const scoreArrayAmbiguous: number[][] = scoreArray.map(
           (x: number, index: number) => [x, index, Math.abs((predictedScore - x) / predictedScore)]).filter(
-          (x: number[]) => ((x[2] < ambiguousClosenessThreshold) && (x[2] >= 0))).map( // ---- NOTE ---- >= for including the top-score one.
+          (x: number[]) => ((x[2] < ambiguousClosenessThreshold) && (x[2] >= 0))).filter( // ---- NOTE ---- >= for including the top-score one.
```
Comment (joshgummersall) [inline packages/orchestratorlib/src/utility.ts:4392]: If you write this line as `([, , someDescriptiveLabel]: number[]) => ((someDescriptiveLabel < ambiguousClosenessThreshold) && (someDescriptiveLabel >= 0))).filter(` it's a little more self-documenting.

**Review by munozemilio — APPROVED** [review]

**Comment by munozemilio** [comment]
@hcyang Ill approve and let you merge so you can address @joshgummersall comment

File: packages/orchestratorlib/src/utility.ts (hunk 1 above)
Comment (hcyang) [reply packages/orchestratorlib/src/utility.ts:4392]: sure, will make them more descriptive

**Review by joshgummersall — APPROVED** [review]
Thanks!

**Pull request merged**